Formula/mdcat.rb

```diff
@@ -22,8 +25,20 @@ class Mdcat < Formula
     depends_on "pkg-config" => :build
   end
 
+  resource "asciidoc" do
+    url "https://files.pythonhosted.org/packages/8a/57/50180e0430fdb552539da9b5f96f1da6f09c4bfa951b39a6e1b4fbe37d75/asciidoc-10.2.0.tar.gz"
+    sha256 "91ff1dd4c85af7b235d03e0860f0c4e79dd1ff580fb610668a39b5c77b4ccace"
+  end
+
   def install
     system "cargo", "install", *std_cargo_args
+    # create virtualenv for generating manpage
+    venv_root = libexec/"venv"
+    venv = virtualenv_create(venv_root, "python3")
+    venv.pip_install resource("asciidoc")
+    system "#{venv_root}/bin/a2x", "-L", "--doctype", "manpage", "--format", "manpage", "mdcat.1.adoc"
+    man1.install "mdcat.1"
+    rm_rf venv_root    
```

Comment:
Hmm, used `github` web ui to make the changes. Somehow, this additional crept in.